package.json

```diff
@@ -12,48 +12,26 @@
     "cookie-parser": "~1.3.3",
     "deasync": "0.1.0",
     "debug": "~2.0.0",
+    "dreamcatcher-client": "git+ssh://git@github.com/LeisureLink/dreamcatcher-client.git#master",
+    "env-configurator": "git+ssh://git@github.com/LeisureLink/env-configurator.git#develop",
     "errorhandler": "^1.3.0",
     "express": "~4.9.0",
     "express-validator": "^2.7.0",
-    "express-winston": "^0.2.9",
     "express-xml-bodyparser": "0.0.4",
     "js2xmlparser": "^0.1.7",
+    "lla-mapper": "git+ssh://git@github.com/LeisureLink/lla-mapper.git#master",
+    "lla-logger": "git+ssh://git@github.com/LeisureLink/lla-logger.git#master",
     "method-override": "^2.3.0",
     "morgan": "^1.5.0",
     "multer": "^0.1.6",
     "request": "^2.53.0",
     "serve-favicon": "~2.1.3",
     "underscore": "^1.7.0",
-    "winston": "^0.8.3",
-    "xml2js": "^0.4.5",
-    "env-configurator": "git+ssh://git@github.com/LeisureLink/env-configurator.git#develop"
+    "xml2js": "^0.4.5"
```

Comment:
I think it was just that it takes forever to npm install and I believe Devin was having some issues installing it. But I guess that’s not too bad, so it would be fine to leave in

From: WWGray4 [mailto:notifications@github.com]
Sent: Thursday, July 30, 2015 9:17 AM
To: LeisureLink/adaptor-boilerplate
Cc: Fitzgerald, Matt
Subject: Re: [adaptor-boilerplate] Refactored boilerplate app to meet new standards (#5)


In package.json<https://github.com/LeisureLink/adaptor-boilerplate/pull/5#discussion_r35888842>:

>      "method-override": "^2.3.0",

>      "morgan": "^1.5.0",

>      "multer": "^0.1.6",

>      "request": "^2.53.0",

>      "serve-favicon": "~2.1.3",

>      "underscore": "^1.7.0",

> -    "winston": "^0.8.3",

> -    "xml2js": "^0.4.5",

> -    "env-configurator": "git+ssh://git@github.com/LeisureLink/env-configurator.git#develop"

> +    "xml2js": "^0.4.5"

What issues do you recall? I can't remember anything specific with this library, so I'm not sure what we need to look for.

Also, I could see a case for removing it, as xml conversion functionality definitely won't be necessary for all/maybe even most adaptors. (Though I suppose the boilerplate philosophy is to include more and have adaptors remove what they don't need later)

—
Reply to this email directly or view it on GitHub<https://github.com/LeisureLink/adaptor-boilerplate/pull/5/files#r35888842>.